dark/index.html

```diff
@@ -63,11 +70,86 @@
 
     <!-- Favicon -->
 
-    <link rel="icon" type="image/png" href="assets/img/Yoshitha Rathnayake 2.png" alt="Yoshitha Rathnayake" />
+    <link
+      rel="icon"
+      type="image/png"
+      href="assets/img/Yoshitha Rathnayake 2.png"
+      alt="Yoshitha Rathnayake"
+    />
 
     <!-- Title -->
 
     <title>YOSHITHA RATHNAYAKE</title>
+
+    <style>

```

Comment:
Great Job! @william-monroy but some changes are requested, Can you put these internel css to the externel css file assets/css/styles.css

Reply:
Sure! :octocat:

Reply:
Can you please remove the styles in the light/index.html file also @william-monroy